<launch>
  <node pkg="velo2cam_calibration" type="velo2cam_calibration" name="velo2cam_calibration" output="screen">
    <remap from="~cloud1" to="/laser_pattern/centers_cloud"/>
    <remap from="~cloud2" to="/stereo_pattern/centers_cloud"/>
    <remap from="~cloud3" to="/velodyne_points"/>
    <param name="radius" value="0.12"/>
    <param name="centers_distance" value="0.3"/>
    <param name="sync_iterations" value="false"/>
  </node>
</launch>
